workflows/getting_started/edge_detection.py

```diff
@@ -0,0 +1,28 @@
+from urllib import request
+
+import cv2
+import flytekit
+from flytekit import task, workflow
+from flytekit.types.file import FlyteFile
+
+@task
+def edge_detection_canny(image_location:str) -> FlyteFile:
```

Comment:
let us not have an edge detection example. This makes the dockerfile bloated. Lets just have a simple example that prints hello world - or sum of squares or something

Reply:
updated